mobwrite/mobwrite_server.py

```diff
@@ -227,7 +227,8 @@ def do_POST(self):
     data = data[div + 2:]
     data = urllib.unquote(data)
     self.send_response(200)
-    self.send_header('Content-type','text/plain')
+    self.send_header('Content-type', 'text/plain')
+    self.send_header('Access-Control-Allow-Origin', '*')
```

Comment:
Ought this not to be limited to `example.codecity.world` and subdomains thereof?